website/src/categories/components/checkbox/usage.md

```diff
@@ -6,27 +6,325 @@ eleventyNavigation:
+  </figcaption>
+</figure>
+
+</section>
+
+</section>
+
+<section>
+
+## Options
+
+With these options, you can tweak the appearance of the radio in Figma. They are available in the Design Panel so you can compose the checkbox to exactly fit the user experience need for the use case you are working on.
+
+{{ 'components/checkbox/sl-checkbox-figma-options.svg' | svgImage }}
+
+|Item|Options|Description|
+|-|-|-|
+|Size|`'md' (Default), 'lg'`|The checkbox and the label element come in two sizes: medium (default) and large, providing flexibility to align with your design requirements.|
+|Variant|`'default', 'invalid', 'valid'`|This variants are provide to give feedback to the user about the interaction o the necessity of another action more.|
+|States|`'idle', 'hover', 'active', 'disabled'`|Checkboxes interactive states to replicate the real behavior into your UI.|
+|Status |`'Checked', 'Unchecked'`|To determinate between selected and unselect version.|
+|Indeterminate |`'on', 'off'`|To turn the indeterminate state on or off.|
+|Label|`'on', 'off'`|To turn to label on or off (label is turned on by default)|
+|Label text|`'input text'`|To insert the text of the label.|
+|Focus ring|`'on', 'off'`|Turn the focus ring option to show the focus state of the checkbox.|
+
+{.ds-table .ds-table-align-top}
+
+</section>
+
 <section>
 
-## Usage
+<div class="ds-example">
+
+<sl-label for="group">Checkbox group</sl-label>
+<sl-checkbox-group id="group" hint="Pick one of these options.">
+<sl-checkbox>Check me</sl-checkbox>
+<sl-checkbox>No me</sl-checkbox>
+<sl-checkbox>I was here first!</sl-checkbox>
+<sl-checkbox disabled>Can't check me, even if you wanted to</sl-checkbox>
+</sl-checkbox-group>
+
+</div>
+
+<div class="ds-code">
+
+  ```html
+  <sl-label for="group">Checkbox group</sl-label>
+  <sl-checkbox-group id="group" hint="Pick one of these options.">
+    <sl-checkbox>Check me</sl-checkbox>
+    <sl-checkbox>No me</sl-checkbox>
+    <sl-checkbox>I was here first!</sl-checkbox>
+    <sl-checkbox disabled>Can't check me, even if you wanted to</sl-checkbox>
+  </sl-checkbox-group>
+  ```
 
-Lorem ipsum dolor sit amet
-Consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus. Sed dignissim, metus nec fringilla accumsan, risus sem sollicitudin lacus, ut interdum tellus elit sed risus. Maecenas eget condimentum velit, sit amet feugiat lectus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Praesent auctor purus luctus enim egestas, ac scelerisque ante pulvinar. Donec ut rhoncus ex. Suspendisse ac rhoncus nisl, eu tempor urna. Curabitur vel bibendum lorem. Morbi convallis convallis diam sit amet lacinia. Aliquam in elementum tellus.
+</div>
 
 </section>
 
 <section>
 
-## How to use
+## Checkbox group
```

Comment:
this is actually part of the anatomy. Wouldn't it be better to maybe combine the checkbox and checkbox group in 1 single image and 1 single table?

Reply:
@arecuenco-dsgn @DanielleRameau what is your opinion here? :)

Reply:
Yes, I agree. I think that's easier to understand. Maybe call 'checkboxes' 'checkbox group' instead. This way we can still explain what the checkbox input is and what the behavior of the checkbox group is. @arecuenco-dsgn do you agree?

Reply:
Sorry, but I don't think the same. It's better to explain the checkbox alone to make clear that it's a single component and describe all that matters for this. Then, you can introduce the checkbox group and define its elements.

With this structure, we move from the small part to the big part (group). What do you think?

@DanielleRameau, @Diaan, @anna-lach

Reply:
I still think is makes it messy and too different from other components if we have too many separate headings for separate component parts in one file. I think the anatomy is actually the easiest part where we can merge the two; you can still make clear what property belongs to which component, but also you can easily explain the relations between them without needing to repeat yourself.

Reply:
Anatomy is updated and the checkbox group section from the `usage` tab is removed. :)

Reply:
Great! Then we can merge, right? 👯